### PR TITLE
K arabia

### PR DESCRIPTION
**Pull request by TinWiz**
Edited script to make Rashidun caliphate more successful initially with conversions of Arabs, and to add the kingdom of Arabia as a de jure kingdom.

### CODE REVIEW

File: common/on_action/TFE_islamic_on_actions.txt [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# IRToCK3: Last updated for CK3 1.7.2
-
-on_game_start = {
+﻿on_game_start = {
 	on_actions = {
```
Comment (IhateTrains) [inline common/on_action/TFE_islamic_on_actions.txt:2]: The the CK3 version comment got removed.

File: common/on_action/TFE_title_on_actions.txt [hunk 2]
```diff
@@ -1,6 +1,4 @@
-# IRToCK3: Last updated for CK3 1.7.2
-
-#On actions about titles
+﻿#On actions about titles
```
Comment (IhateTrains) [inline common/on_action/TFE_title_on_actions.txt:2]: ditto

File: common/on_action/TFE_title_on_actions.txt [hunk 3]
```diff
@@ -103,6 +106,9 @@ TFE_on_title_gain = {
 					set_de_jure_liege_title = title:e_rashidun
 				}
 			}
+			title:k_arabia = {
```
Comment (IhateTrains) [inline common/on_action/TFE_title_on_actions.txt]: Depending on converter's title mappings, `k_arabia` may be missing. Maybe wrap this in an
```
if = {
	limit = {
		exists = title:k_arabia
	}
	<effects>
}
```
block?

**Review by IhateTrains — CHANGES_REQUESTED** [review]

File: common/on_action/TFE_title_on_actions.txt (hunk 3 above)
Comment (TinWiz) [reply common/on_action/TFE_title_on_actions.txt]: Sure, done.

**Review by IhateTrains — APPROVED** [review]

**Pull request merged**